packages/liveblocks-client/src/LiveList.ts

```diff
@@ -20,15 +20,17 @@ type LiveListItem = [crdt: AbstractCrdt, position: string];
  */
 export class LiveList<T> extends AbstractCrdt {
   // TODO: Naive array at first, find a better data structure. Maybe an Order statistics tree?
-  #items: Array<LiveListItem> = [];
+  private _items: Array<LiveListItem>;
```

Comment:
Minor nit: can we settle on using a consistent prefix? I think a `_` prefix works just fine, but noticed a double underscore `__` used in the other file.

Reply:
The double `__` is ugly :/ 

There are collisions with some getters in `AbstractCrdt` and I didn't find a better prefix. 

Maybe there is a way to get rid of those protected getters but for the scope of the PR, I just wanted to remove babel extra code. I can create another task to find a better pattern for the base class to remove these double underscore for good.

Reply:
> _"There are collisions with some getters in AbstractCrdt and I didn't find a better prefix."_

Aha, I see. That wasn't clear to me at first — thanks for clarifying 👍